Calling/ClientApp/src/containers/TypingIndicator.ts

```diff
@@ -0,0 +1,58 @@
+import { connect } from 'react-redux';
```

Comment:
nit: on UI, it is now showing name of the use who is typing. It indicate “is typing” only

Reply:
Created a separate task to track this suggestion.